bin/generate_commit

```diff
@@ -104,6 +104,18 @@ function _create_changelog() {
         --date=iso
 }
 
+function _create_readme(){
+    
+    if [ -e README.md ]; then
+        echo "File README.md already exists!"
+    else
+        echo "iam here"
+        echo "### Title1 $1" > README.md
```

Comment:
I recommend putting these variables at the top of the function in a local variable for readability:

```
local title=$1
local summary=$2
...
```